module/swig/php/Makefile.am

```diff
@@ -24,7 +24,6 @@ AM_CFLAGS = -fexceptions \
 	${EXTRACFLAGS} \
 	${LIBUSB_CFLAGS} \
 	${PTHREAD_CFLAGS} \
-	${LIBAVAHI_CFLAGS} \
```

Comment:
You don't want to remove this. Since ow.h (or whichever public OW header it was) uses avahi-client, this will fail if avahi-client.h is in i.e. /usr/local/include (i.e. FreeBSD) which is not default search path.

Reply:
This comes from #4, maybe @heinervdm can explain why he introduced this change.